charts/app-config/values-production.yaml

```diff
@@ -2,16 +2,16 @@
 # individual apps below.
 
 govukEnvironment: production
-externalDomainSuffix: gov.uk
-publishingDomainSuffix: publishing.service.gov.uk
-assetsDomain: assets.publishing.service.gov.uk
+externalDomainSuffix: production.publishing.service.gov.uk
+publishingDomainSuffix: production.publishing.service.gov.uk
+assetsDomain: assets.production.publishing.service.gov.uk
```

Comment:
Whilst I think these are valid domains - should we be using just `publishing.service.gov.uk`, without "production" subdomain?

Reply:
Oof, good catch, thanks! I'd been lulled into a false sense of security from doing the integration->staging one earlier.

Reply:
Think I've found and fixed 'em all. PTAL. Thanks!